templates/vdc/vdc.py

```diff
@@ -61,12 +62,19 @@ def account(self):
         """
         An account getter
         """
+        if self._account is not None:
```

Comment:
what happens if the openvcloud gets updated and we have connection in cache?
Same for the `openvcloud` property

Reply:
the same idea crossed my mind, and i didn't find a better solution (for now) but doing it without caching is pretty intensive, on the robot. some getters will then have to do a service find every time they are accessed

Reply:
one way to solve this is to have the openvcloud connection directly on the schema of the vdc service But then we cut the dependency chain a bit :-s

Reply:
I hated that in ays, that we need to specify both the account and the connection for a vdc to work :/ this looks ugly and stupid at the same time.

Also i don't know how to make 0-robot support this `parent` look up without introducing `parenting` and all this hassle that we tried to avoid in 0-robot